packages/dapp/pages/_app.tsx

```diff
@@ -5,6 +5,7 @@ import "./styles/dark-light-mode-toggle.css";
 
 import { AppProps } from "next/app";
 import Head from "next/head";
+import Script from "next/script"; //@note Fix: (next/script warning)
```

Comment:
You might not need to prefix `@note` because that doesn't seem to be a standard that I've seen in UI code.

Reply:
I've used this extension that highlights such tags like `@note` `@todo` etc 
https://marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks
@pavlovcik